docs/guides/index.mdx

```diff
@@ -92,7 +90,7 @@ Beyond Qiskit and its related packages is the [Qiskit ecosystem](https://www.ibm
     <Card
       title="Error code registry"
       description="Look up error codes and suggested solutions."
-      href="/docs/errors"
+      href="https://quantum.cloud.ibm.com/docs/errors"
```

Comment:
I changed this to the full URL since it might be broken ... clicking on this link 404s in both PR preview and local preview. Will check with @arnaucasau and @Eric-Arellano